INSTALL

```diff
@@ -1,288 +1,368 @@
-Since the dynamic libraries have been copied to /local/lib/save_dynamic before
-deletion, you can copy them back to /local/lib. If the worst comes to the worst, you
-must re-install the optional packages.
-
-*********************
-****     SCIP    ****
-*********************
-
-To build SCIP:
-
-* download the scipoptsuite at http://scip.zib.de/ . 
-
-   Notice that SCIP
-   is not distributed under GPL, but the ZIB Academic License
-   (http://scip.zib.de/academic.txt).
-   
-* Unpack it and then compile it with
- 
-   make ZLIB=false GMP=false READLINE=false scipoptlib
-   
-   (To compiling scip for 32bit systems, add also ARCH=x86 to the make.)
-   
-   If your C++ compiler is too old, you may need to add
-   CXXFLAGS="-std=c++0x -fPIC"
-   
-* IMPORTANT: ** Don't try to build scipopt as a shared library!  It does not work
-      because their build system is broken **
-      
-For configure you must set
-      
-   --with-scipoptsuite-src=DIR
-   
-to name the directory where SCIP can be found.
-
-
-********************
-**** plain MAKE ****
-********************
-
-Under Linux one can still use our old simple plain Makefile.
-
-IT IS MEANT FOR THE DEVELOPMENT OF NORMALIZ and the creation of static binaries under Linux.
-
-This Makefile is mainly used for development and debugging. It builds statically linked binaries
-with -g (can be changed in Makefile.configuration).
-
-It is assumed that the optional libraries are installed in
-
-   local
-   
-within the Normaliz directory.
-
-1) Go to the source directory
-
-2) Check the configuration in Makefile.configuration, especially the locations of
-the optional packages and their versions.
-
-3) Say 
-
-        make -f Makefile.classic
-
-It should work without change of Makefile.classic or Makefile.configuration on many systems
-that have a system installation of the needed libraries. (We must use the name Makefile.classic since Makefile is occupied by autotools.) 
-
-Without further arguments, Makefile.classic will build Normaliz with OpenMP and CoCoALib, but without SCIP and Flint (see below how to change this behavior)
-
-If everything is successful, then the executable file normaliz is
-in the source directory. You may of course want to move them somewhere else.
-
-
-Makefile.classic accepts optional arguments (also simultaneously):
-
-        make -f Makefile.classic OPENMP=no  # deactivates OpenMP
-        make -f Makefile.classic COCOA=no   # deactivates CoCoALib
-        make -f Makefile.classic SCIP=yes   # compile with SCIP
-        make -f Makefile.classic Flint=no   # deactivates Flint
-        make -f Makefile.classic EANTIC=no  # deactivates e-antic
-        make -f Makefile.classic NMZ_DEVELOP=no  # deactivates development version
-
-You must make sure that everything is compiled with the same set of options!
-In doubt use "make -f Makefile.classic clean" beforehand.
-
-Similarly
-
-        make -f Makefile.classic maxsimplex
```

Comment:
Is this different from `cd source; make maxsimplex/maxsimplex` with autotools?

Reply:
I would like to keep both possibilities, via autotools and via Makefile.classic

Reply:
It is of course possible to keep Makefile.classic on the private side and not mention it in INSTALL.  Sp you can remove it from INSTALL as well as from the distribution via EXTRADIST. But I would like to keep it in the git repository.

Reply:
Sounds good. If it's useful to you we should keep it in the repo.